Assets/GothicVR/Scripts/Creator/WaynetCreator.cs

```diff
@@ -11,10 +11,13 @@ public static class WaynetCreator
         public static void Create(GameObject root, WorldData world)
         {
             var waynetObj = new GameObject(string.Format("Waynet"));
-            waynetObj.transform.parent = root.transform;
 
+            DijkstraPathFinder pathFinder = waynetObj.AddComponent<DijkstraPathFinder>();
+            waynetObj.transform.parent = root.transform;
 
             CreateWaypoints(waynetObj, world);
+            pathFinder.SetDijkstraWaypointsOriginal(world); // Has to be here to get the waypoints position
```

Comment:
This can be moved to somewhere else BUT it can't be before the above CreateWaypoints method runs. I think this is the place for this so that any class at any time can use pathfinding. Happy to be proven wrong though!